relayer/src/link/relay_path.rs

```diff
@@ -501,15 +511,15 @@ impl<ChainA: ChainHandle, ChainB: ChainHandle> RelayPath<ChainA, ChainB> {
         initial_od: OperationalData,
     ) -> Result<S::Reply, LinkError> {
         // We will operate on potentially different operational data if the initial one fails.
+        let span = span!(Level::INFO, "relay", id = %initial_od);
```

Comment:
```suggestion
        let span = span!(Level::INFO, "relay", id = %initial_od.tracking_id());
```

Reply:
Note that this log provides a bit more context beyond the tracking identifier, namely:

> 2021-12-22T12:18:40.359804Z DEBUG ThreadId(444) relay{id=BS6CBKeio4 ->Destination @0-25087; len=1}: [ibc-0:transfer/channel-0 -> ibc-1] delayed by: 290.181236ms [try 1/5].

- it also shows whether the relay operation is targeting the Source or Destination (`->Destination` in this case)
- and the height where clients proofs are built `0-25087`
- and the # of messages the operation comprises.

I think we do want all of this information in there.
Should we structure it more clearly with KVs?

Reply:
In 85c60fa8f1b37e0f823aaa3cf03c79bafd8ef994, I have opted to use the `odata` key to display the operational data information without commingling with `id` used just for tracking IDs. Here and in the other places.